arithmetization/src/main/java/net/consensys/linea/zktracer/module/ecdata/EcData.java

```diff
@@ -89,18 +89,16 @@ public void callEcData(
     operations.add(ecDataOperation);
 
     switch (ecDataOperation.precompileFlag()) {
-      case PRC_ECADD -> ecAddEffectiveCall.addPrecompileLimit(
-          ecDataOperation.internalChecksPassed() ? 1 : 0);
-      case PRC_ECMUL -> ecMulEffectiveCall.addPrecompileLimit(
-          ecDataOperation.internalChecksPassed() ? 1 : 0);
-      case PRC_ECRECOVER -> ecRecoverEffectiveCall.addPrecompileLimit(
+      case PRC_ECADD -> ecAddEffectiveCall.addLimit(ecDataOperation.internalChecksPassed() ? 1 : 0);
+      case PRC_ECMUL -> ecMulEffectiveCall.addLimit(ecDataOperation.internalChecksPassed() ? 1 : 0);
+      case PRC_ECRECOVER -> ecRecoverEffectiveCall.addLimit(
           ecDataOperation.internalChecksPassed() ? 1 : 0);
       case PRC_ECPAIRING -> {
         // ecPairingG2MembershipCalls case
         // NOTE: the other precompile limits are managed below
         // NOTE: see EC_DATA specs Figure 3.5 for a graphical representation of this case analysis
         if (!ecDataOperation.internalChecksPassed()) {
-          ecPairingG2MembershipCalls.addPrecompileLimit(0);
+          ecPairingG2MembershipCalls.addLimit(0);
```

Comment:
`addLimit` seems like a misnomer. For count only operations are updating a tally, so `updateTally` seems more appropriate.